Sources/Core/Microsoft.StreamProcessing/Utilities/Config.cs

```diff
@@ -617,6 +623,14 @@ public ConfigModifier CodeGenAfa(bool value)
             return this;
         }
 
+        public ConfigModifier SerializationCompressionLevel(SerializationCompressionLevel value)
+        {
+            this.modifications.Add(GatedModification<SerializationCompressionLevel>.Create(
+                value,
+                v => { var old = Config.SerializationCompressionLevel; Config.SerializationCompressionLevel = v; return old; }));
```

Comment:
If we are multi-lining the lambda in the other change, please do so here as well.